static/js/pages/committee-single.js

```diff
@@ -218,101 +217,145 @@ $(document).ready(function() {
     var query = {cycle: cycle};
     var path;
     switch ($table.attr('data-type')) {
-      case 'committee-contributor':
-        path = ['committee', committeeId, 'schedules', 'schedule_a', 'by_contributor'];
-        tables.initTableDeferred($table, null, path, query, committeeColumns, aggregateCallbacks, {
-          dom: tables.simpleDOM,
-          order: [[1, 'desc']],
-          pagingType: 'simple',
-          lengthChange: false,
-          pageLength: 10,
-          useHideNull: false,
-          hideEmpty: true
-        });
-        break;
-      case 'contribution-size':
-        path = ['committee', committeeId, 'schedules', 'schedule_a', 'by_size'];
-        tables.initTableDeferred($table, null, path, query, sizeColumns, aggregateCallbacks, {
-          dom: 't',
-          order: [[1, 'desc']],
-          pagingType: 'simple',
-          lengthChange: false,
-          pageLength: 10,
-          useHideNull: false,
-          hideEmpty: true
-        });
-        break;
-      case 'receipts-by-state':
-        path = ['committee', committeeId, 'schedules', 'schedule_a', 'by_state'];
-        query = _.extend(query, {per_page: 99, hide_null: true});
-        tables.initTableDeferred($table, null, path, query, stateColumns, aggregateCallbacks,
-          _.extend(
-            {},
-            tableOpts,
-            {
-              dom: 't',
-              order: [[1, 'desc']],
-              paging: false,
-              scrollY: 400,
-              scrollCollapse: true
-            }
-          )
-        );
-        events.on('state.map', function(params) {
-          var $map = $('.state-map');
-          highlightRowAndState($map, $table, params.state, true);
+    case 'committee-contributor':
+      path = ['committee', committeeId, 'schedules', 'schedule_a', 'by_contributor'];
+      tables.DataTable.defer($table, {
+        path: path,
+        query: query,
+        columns: committeeColumns,
+        callbacks: aggregateCallbacks,
+        dom: tables.simpleDOM,
+        order: [[1, 'desc']],
+        pagingType: 'simple',
+        lengthChange: false,
+        pageLength: 10,
+        useHideNull: false,
+        hideEmpty: true
+      });
+      break;
+    case 'contribution-size':
+      path = ['committee', committeeId, 'schedules', 'schedule_a', 'by_size'];
+      tables.DataTable.defer($table, {
+        path: path,
+        query: query,
+        columns: sizeColumns,
+        callbacks: aggregateCallbacks,
+        dom: 't',
+        order: [[1, 'desc']],
+        pagingType: 'simple',
+        lengthChange: false,
+        pageLength: 10,
+        useHideNull: false,
+        hideEmpty: true
+      });
+      break;
+    case 'receipts-by-state':
+      path = ['committee', committeeId, 'schedules', 'schedule_a', 'by_state'];
+      query = _.extend(query, {per_page: 99, hide_null: true});
```

Comment:
Identifier 'per_page' is not in camel case.<br>Identifier 'hide_null' is not in camel case.